Kentor.AuthServices.Tests/WebSSO/AuthServicesUrlsTests.cs

```diff
@@ -38,7 +38,7 @@ public void AuthServicesUrls_Ctor_NullCheckApplicationUrl()
         [TestMethod]
         public void AuthServicesUrls_Ctor_NullCheckModulePath()
         {
-            Action a = () => new AuthServicesUrls(new Uri("http://localhost"), null);
+            Action a = () => new AuthServicesUrls(new Uri("http://localhost"), (string)null);
```

Comment:
I agree that casting of null isn't elegant. It's better to use a named argument:

`new AuthServicesUrls(new Uri("http://localhost"), modulePath: null)`